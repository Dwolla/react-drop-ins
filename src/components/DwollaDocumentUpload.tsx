import type { FC } from 'react';
import { DocumentUploadAttributes } from '../@types/component-attributes';

const DwollaDocumentUpload: FC<DocumentUploadAttributes<any>> = (props) => {
  return <dwolla-customer-create {...props} />;
};

export default DwollaDocumentUpload;
