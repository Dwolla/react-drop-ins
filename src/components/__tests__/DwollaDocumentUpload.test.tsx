import TestRenderer from 'react-test-renderer';
import DwollaDocumentUpload from '../DwollaDocumentUpload';

test('DwollaDocumentUpload', () => {
  const tree = TestRenderer.create(<DwollaDocumentUpload customerId="123" />).toJSON();

  expect(tree).toMatchSnapshot();
});
