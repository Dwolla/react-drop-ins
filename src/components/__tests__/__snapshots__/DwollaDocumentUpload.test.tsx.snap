// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaDocumentUpload 1`] = `
<dwolla-document-upload
  customerId="123"
/>
`;
